projection.tex

```diff
@@ -74,11 +74,138 @@
 %
 \end{abstract}
 
-\section{Projection Method}
 %
 \label{s:projection}
-
-TBA.
+\section{Govern equations for 2D incompressible flow}
+For simplicity, suppose $\rho=1$. \\
+Momentum equation in x-direction:
+\begin{align*}
+\frac{\partial u}{\partial t}+u \frac{\partial u}{\partial x}+v \frac{\partial u}{\partial y}=-\frac{\partial p}{\partial x}+ \nu (\frac{\partial^2 u}{\partial x^2}+\frac{\partial^2 u}{\partial y^2})
+\tag{1}
+\end{align*}
+Momentum equation in y-direction:
+\begin{align*}
+\frac{\partial v}{\partial t}+u \frac{\partial v}{\partial x}+v \frac{\partial v}{\partial y}=-\frac{\partial p}{\partial y}+ \nu (\frac{\partial^2 v}{\partial x^2}+\frac{\partial^2 v}{\partial y^2})
```

Comment:
(In new PR), please use subscript $v_1$ and $v_2$ for the $x_1$ ( $x$ ) and $x_2$ ( $y$ ) axes.